src/app/features/auth-management/auth-management.component.html

```diff
@@ -0,0 +1,5 @@
+<app-init-reset-password *ngIf="mode === 'initResetPassword'" [email]="email" />
+<app-reset-password
+  *ngIf="mode === 'resetPassword'"
+  [email]="email"
+  [oobCode]="oobCode" />
```

Comment:
I've a hard time to understand the split and the involved component names. I mean I can follow the breadcrumbs to understand what's happening but it doesn't fell natural or simple.

The naming with init is also confusing. It's short for initiate? Because init is very ofsten used for initialize. Maybe that's part of the confusion?

Also seeing that clear split of views, I think your simulating a think you router should do: depending on a given keyword/param/url segment show different views? `password` and `password/reset`

Reply:
I'm limited here by how firebase handles different authentication tasks. They require you to manage all of them (resetting Password, confirming Email etc.) through the same path, just with different query parameters. That's why the component is structured like this.